code/drasil-utils/lib/Utils/Drasil/Document.hs

```diff
@@ -11,14 +11,6 @@ blank = text ""
 indent :: Doc -> Doc
 indent = nest 4
 
--- | Helper which concatenates two Docs.
-(+:+) :: Doc -> Doc -> Doc
-a +:+ b = a <> b
-
--- | Helper which concatenates two Docs and appends a period.
-(+:+.) :: Doc -> Doc -> Doc
-a +:+. b = a +:+ b <> text "." 
```

Comment:
Except that you went too far! I did want this particular combinator to be created.

Reply:
If I add the following combinator, then I will also need to add `+:+`, as I had in the previous PR.

```
-- | Helper which concatenates two Docs and appends a period.
(+:+.) :: Doc -> Doc -> Doc
a +:+. b = a +:+ b <> text "." 
```

Or would you prefer the following for now.
```
-- | Helper which concatenates two Docs and appends a period.
(+:+.) :: Doc -> Doc -> Doc
a +:+. b = a <> b <> text "." 
```

Reply:
I prefer the 2nd one.

Reply:
The PR is now updated to reflect the requested change.